### ISSUE TITLE
Stop event should be assigned task ID

### ISSUE THREAD

**Issue by JackOfAllTrades4111** [mathisdt/trackworktime#98]
As parallel time recording for different tasks is allowed, the stop events should be assigned a task ID as well. Otherwise, the report data are ambiguous as to which stop event belongs to which start event. 
Ideally, if there is only one start event, the default task ID should be that of the running task.

**Comment by mathisdt** [comment]
That is not quite correct, parallel time recording is **not** supported in TWT. Every new start event automatically stops the period before so there's always only one task active. That reflects the circumstance that we all only can do one thing at a time - either we work on project A or on project B. (If there are situations when you work on multiple things at the same time, please create a special task for that, but I expect that to be the rare exception.)
For this reason, the stop event intentionally does not support selecting a task. It always ends the task which was active during the last period, which means the task selected in the last start event.

**Comment by JackOfAllTrades4111** [comment]
> parallel time recording is not supported in TWT. Every new start event automatically stops the period before so there's always only one task active.

Definetely not the observed behavoiur for 1.0.6 under Android 9. When I trigger a second start event from either the running or another task, I just get another start event.

Apart from that, recording two tasks in parallel may actually be desired (not even implying multitasking, but, e.g., recording work on the same project on different accounts, etc.) - having the stop event write out the task ID as such would thus not do any harm, but at most extend functionality.

**Comment by mathisdt** [comment]
Absolutely right, there is no auto-generated stop event. But logically, every new start event stops the period before (if any). This can be observed in the export where the sums for every task are printed.

**Comment by JackOfAllTrades4111** [comment]
Well, I guess then it would be nice if stop events were (at least during report generation) generated and printed to the 'None (all events)' type of report, for consistency with the other types of report.
Then, in turn, you may also add the task ID to which the stop event refers.
